src/main/tv/codely/scala_intro_examples/lesson_07_generics/exercise_solutions/joan_miralles/GenericFryer.scala

```diff
@@ -0,0 +1,7 @@
+package tv.codely.scala_intro_examples.lesson_07_generics.exercise_solutions.joan_miralles
+
+import scala.concurrent.Future
+
+trait GenericFryer[IngredientType, FriedIngredientType] {
```

Comment:
Good one!

However, we should try to avoid specifying these kind of prefixes and suffixes. I'm referring to the `GenericXxx` or `XxxInterface` ones. They doesn't bring any value so we can get rid of them 🙂